nemo/lightning/nemo_logger.py

```diff
@@ -158,7 +157,8 @@ def _setup_trainer_loggers(self, trainer, dir, version):
             for logger in trainer.loggers:
                 if isinstance(logger, TensorBoardLogger):
                     logger._version = version or ""
-                    logger._root_dir = Path(dir) / logger.save_dir
+                    rel_save_dir = os.path.relpath(logger.save_dir)
+                    logger._root_dir = Path(dir) / rel_save_dir
```

Comment:
Could change to single line here unless rel_save_dir is needed later on - 

logger._root_dir = Path(dir) / os.path.relpath(logger.save_dir)